games-strategy/7k-ambition/7k-ambition-3.2.4.ebuild

```diff
@@ -47,6 +47,15 @@ src_unpack() {
 }
 
 src_prepare() {
+	if use multiplayer ; then
+		gcc --version | grep 12\. || \
```

Comment:
btw you should use `if tc-is-gcc && ver_test $(gcc-version) -ne 12`  instead and add `BDEPEND="multiplayer? ( sys-devel/gcc:12 )"` and also made a new revision